test/verify/check-services

```diff
@@ -188,6 +188,268 @@ WantedBy=default.target
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        b.wait_in_text(svc_sel('hourly_timer.timer'), "Hourly timer")
+        self.assertIn("Fri 2020-04-10 03:05", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-04-10 03:07:00'")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        self.assertIn("Fri 2020-04-10 03:26", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-04-10 04:00:00'")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        # checks if timer runs on next hour at 5 min and 26 min
+        self.assertIn("Fri 2020-04-10 04:05", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("timedatectl set-time '2020-04-10 04:10:00'")
+        b.wait_present(svc_sel('hourly_timer.timer'))
+        self.assertIn("Fri 2020-04-10 04:26", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("systemctl stop hourly_timer.timer")
+        # creates a new timer that runs at today at 23:59
+        b.wait_visible("#create-timer")
+        b.click('#create-timer')
+        b.wait_popup("timer-dialog")
+        b.set_val("#servicename", "no_repeat_timer")
+        b.set_val("#description", "No repeat timer")
+        b.set_val("#command", "/bin/sh -c 'while true; do sleep 5; done'")
+        b.click("#boot-or-specific-time button")
+        b.click("#boot-or-specific-time .dropdown-menu li[value=2] a")
+        b.set_val("#hr", "23")
+        b.set_val("#min", "59")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('no_repeat_timer.timer'))
+        b.wait_in_text(svc_sel('no_repeat_timer.timer'), "No repeat timer")
+
+        m.execute("timedatectl set-time '2020-04-10 04:10:00'")
+        m.execute("systemctl start no_repeat_timer.timer")
+        b.wait_present(svc_sel('no_repeat_timer.timer'))
+        self.assertIn("Fri 2020-04-10 23:59", m.execute("LC_ALL=C systemctl list-timers"))
+        m.execute("systemctl stop no_repeat_timer.timer")
+        # creates a boot timer that runs after 10 sec from boot
+        b.wait_visible("#create-timer")
+        b.click('#create-timer')
+        b.wait_popup("timer-dialog")
+        b.set_val("#servicename", "boot_timer")
+        b.set_val("#description", "Boot timer")
+        b.set_val("#command", "/usr/bin/sh -c 'echo hello >> /tmp/hello'")
+        b.click("#boot-or-specific-time button")
+        b.click("#boot-or-specific-time .dropdown-menu li[value=1] a")
+        b.set_val("#boot-time", "10")
+        b.click("#timer-save-button")
+        b.wait_popdown("timer-dialog")
+        b.wait_present(svc_sel('boot_timer.timer'))
+        m.execute("systemctl enable boot_timer")
+
+        m.spawn("sync && sync && sync && sleep 0.1 && reboot", "reboot")
+        m.wait_reboot()
+        m.start_cockpit()
+        self.assertIn("hello", m.execute("cat /tmp/hello"))
```

Comment:
The timer runs 10 seconds after boot, but here you assert that the text is present after cockpit started. To avoid a race condition, you should probably safely wait for the text to appear.
Maybe something like this would work:
`m.execute("while [ ! -f /tmp/hello ] ; do sleep 0.5; done")`